src/commands/client-list.json

```diff
@@ -35,6 +35,10 @@
             [
                 "8.0.0",
                 "Replaced `master` `TYPE` with `primary`. `master` still supported for backward compatibility."
+            ],
+            [
+                "8.1.0",
+                "Added filters like USER, ADDR, LADDR, SKIPME, and MAXAGE"
```

Comment:
Good, I forgot about this one. The word "like" sounds like it's some examples but it's actually a complete list.

```Suggestion
                "Added filters USER, ADDR, LADDR, SKIPME, and MAXAGE"
```

Reply:
I realize now that we have a change to ID of CLIENT KILL that we should update in client-kill.json.

It now accepts multiple IDs. Before, it only accepted one id. We should mention that in the history and add `"multiple": true` for the ID argument.

Reply:
@zuiderkwast missed this comment before. Have addressed it now. thank you